DataFormats/Phase2TrackerDigi/src/Phase2ITQCore.cc

```diff
@@ -0,0 +1,162 @@
+#include "DataFormats/Phase2TrackerDigi/interface/Phase2ITQCore.h"
+#include <cmath>
+#include <vector>
+#include <iostream>
+
+//4x4 region of hits in sensor coordinates
+Phase2ITQCore::Phase2ITQCore(int rocid,
+                             int ccol_in,
+                             int qcrow_in,
+                             bool isneighbour_in,
+                             bool islast_in,
+                             const std::vector<int>& adcs_in,
+                             const std::vector<int>& hits_in) {
+  rocid_ = rocid;
+  ccol_ = ccol_in;
+  qcrow_ = qcrow_in;
+  isneighbour_ = isneighbour_in;
+  islast_ = islast_in;
+  adcs_ = adcs_in;
+  hits_ = hits_in;
+}
+
+//Takes a hitmap in sensor coordinates in 4x4 and converts it to readout chip coordinates with 2x8
+std::vector<bool> Phase2ITQCore::toRocCoordinates(std::vector<bool>& hitmap) {
+  std::vector<bool> ROC_hitmap(16, false);
+
+  for (size_t i = 0; i < hitmap.size(); i++) {
+    int row = std::floor(i / 4);
```

Comment:
`std::floor` is not needed for integer division, it already round to the lowest integer value.

Reply:
Thank you.